main.py

```diff
@@ -0,0 +1,41 @@
+import pygame
+from minefield import *
+
+pygame.init()
+mixer.init()
+
+screen = pygame.display.set_mode([Minefield.WINDOW_DIM, Minefield.WINDOW_DIM])
+pygame.display.set_icon(Tile.SPR_MINE)
+
+m = Minefield(screen)
+m.init_field()
+
+running = True
+
+while running:
+
+    ev = pygame.event.get()
+
+    for event in ev:
+        if(event.type == pygame.QUIT):
```

Comment:
Instead of having a bunch of if elses create a dict of options with your event types as keys and function references as values. This dict can then exist outside of main and be imported so you have better separation of responsibilities between classes